src/Decoder.h

```diff
@@ -149,6 +149,7 @@ namespace e57
       double offset_;
       unsigned bitsPerRecord_;
       RegisterT destBitMask_;
+      static const size_t registerBits_ = sizeof(RegisterT) * 8;
```

Comment:
To be consistent with (most of?) the rest of code, this should be `static constexpr` and should not use the `_` suffix.

I see that things are not consistent throughout, however with capitalization. I would prefer:

```cpp
static constexpr size_t RegisterBits = sizeof(RegisterT) * 8;
```

What do you think?

Reply:
Yes, makes sense.